themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties

```diff
@@ -281,3 +281,9 @@ noCertificate=[Pas de certificat]
 
 pageNotFound=Page non trouv\u00e9e
 internalServerError=Une erreur interne du serveur s''est produite
+
+# Identity provider
+idp-email-verification-display-name=Vérification du courriel
+idp-email-verification-help-text=Lier votre compte en validant votre courriel.
+idp-username-password-form-display-name=Nom d''utilisateur et mot de passe
+idp-username-password-form-help-text=Lier votre compte en vous connectant.
```

Comment:
The translation is slightly different than the English source. It should work but just to double check if that is what you meant: `Link your account by login in.`

Reply:
Yup, that's exactly it.

When login in with an  external identity provider, if you already have an account on our keycloak (same email on both side in our case), then you can try to login on our keycloak to prove you own it before linking it to the idp account.

Reply:
+ `Link your account by validating your email.`